lib/sablon/processor/document.rb

```diff
@@ -174,6 +191,9 @@ def consume(allow_insertion)
           when /comment/
             block = consume_block("endComment")
             Statement::Comment.new(block)
+          when /^@([^ ]+):start/
```

Comment:
Could you add a test-case in `documents_test.rb` and a corresponding xml snippet in `test/fixtures/xml`. This helps to understand the required structure of the template.